loadtest/control/utils.go

```diff
@@ -57,15 +57,35 @@ func RandomizeUserName(name string) string {
 }
 
 func emulateUserTyping(t string, cb func(term string) UserActionResponse) UserActionResponse {
+	typingSpeed := time.Duration(100+rand.Intn(20)*10) * time.Millisecond // 100-300ms
+
+	typeCorrect := time.NewTicker(typingSpeed)
+	defer typeCorrect.Stop()
+	typeWrong := time.NewTicker(time.Duration(2+rand.Intn(8)) * typingSpeed)
```

Comment:
Same here, why 2 and 8 ? What do they mean ? Does it mean 2-10 times slower than correct typing speed ? I would refactor that to a separate variable and name it as slowFactor and appropriately comment it to be clear.

Reply:
Actually I made up those numbers. We can document it to be more clear though.